The `default` property of core annotation `@Default` gets stripped when stubbed with kapt

This one is weird. The only annotation affected is `@Default` (all other core annotations appear fine) This could be a bug with [schema-kenerator](https://github.com/SMILEY4/schema-kenerator), [kapt](https://kotlinlang.org/docs/kapt.html) or [avaje-inject](https://github.com/avaje/avaje-inject)

Placing `@Default("xyz")` on any type or property produces the stub (generated by kapt):

```kt
@Default("asc")
enum class OrderBy {
    ASC,
    DESC,
}
```

```java
@kotlinx.serialization.Serializable()
@io.github.smiley4.schemakenerator.core.annotations.Default()
public enum OrderBy {
    // ... remainder omitted for brevity 
```

As you can see, the parameter gets stripped, resulting in:

```
/home/rmcmk/project/build/tmp/kapt3/stubs/main/very/cool/package/names/OrderBy.java:9: error: annotation @Default is missing a default value for the element 'default'
@io.github.smiley4.schemakenerator.core.annotations.Default()
^
```

The only difference I physically see from the other annotations is that `val default: String` is not documented with KDoc, which really shouldn't matter?

[Default.kt](https://github.com/SMILEY4/schema-kenerator/blob/develop/schema-kenerator-core/src/main/kotlin/io/github/smiley4/schemakenerator/core/annotations/Default.kt#L7)
[Description.kt](https://github.com/SMILEY4/schema-kenerator/blob/develop/schema-kenerator-core/src/main/kotlin/io/github/smiley4/schemakenerator/core/annotations/Description.kt#L8)

Wondering if anyone else is using kapt (perhaps from another library than avaje-inject?) having a similar issue?

Edit: It's the name of the parameter... `default` is a restricted java keyword.. duh. Been awhile...

--- a/schema-kenerator-core/src/main/kotlin/io/github/smiley4/schemakenerator/core/annotations/Default.kt
+++ b/schema-kenerator-core/src/main/kotlin/io/github/smiley4/schemakenerator/core/annotations/Default.kt
@@ -5,6 +5,7 @@ import kotlinx.serialization.SerialInfo
 
 /**
  * Specifies a default value for the annotated object.
+ * @param value The default value for the annotated object.
  */
 @OptIn(ExperimentalSerializationApi::class)
 @Target(
@@ -15,4 +16,4 @@ import kotlinx.serialization.SerialInfo
 )
 @SerialInfo
 @Retention(AnnotationRetention.RUNTIME)
-annotation class Default(val default: String)
+annotation class Default(val value: String)

--- a/schema-kenerator-jsonschema/src/main/kotlin/io/github/smiley4/schemakenerator/jsonschema/steps/JsonSchemaCoreAnnotationDefaultStep.kt
+++ b/schema-kenerator-jsonschema/src/main/kotlin/io/github/smiley4/schemakenerator/jsonschema/steps/JsonSchemaCoreAnnotationDefaultStep.kt
@@ -30,7 +30,7 @@ class JsonSchemaCoreAnnotationDefaultStep : AbstractJsonSchemaStep() {
     private fun determineDefault(annotations: List<AnnotationData>): String? {
         return annotations
             .filter { it.name == Default::class.qualifiedName }
-            .map { it.values["default"] as String }
+            .map { it.values["value"] as String }
             .firstOrNull()
     }
 }

--- a/schema-kenerator-swagger/src/main/kotlin/io/github/smiley4/schemakenerator/swagger/steps/SwaggerSchemaCoreAnnotationDefaultStep.kt
+++ b/schema-kenerator-swagger/src/main/kotlin/io/github/smiley4/schemakenerator/swagger/steps/SwaggerSchemaCoreAnnotationDefaultStep.kt
@@ -14,21 +14,21 @@ class SwaggerSchemaCoreAnnotationDefaultStep : AbstractSwaggerSchemaStep() {
 
     override fun process(schema: SwaggerSchema, typeDataMap: Map<TypeId, BaseTypeData>) {
         if (schema.swagger.default == null) {
-            determineDefaults(schema.typeData.annotations)?.also { default ->
+            determineDefault(schema.typeData.annotations)?.also { default ->
                 schema.swagger.setDefault(default)
             }
         }
         iterateProperties(schema, typeDataMap) { prop, propData, propTypeData ->
-            determineDefaults(propData.annotations + propTypeData.annotations)?.also { default ->
+            determineDefault(propData.annotations + propTypeData.annotations)?.also { default ->
                 prop.setDefault(default)
             }
         }
     }
 
-    private fun determineDefaults(annotations: Collection<AnnotationData>): String? {
+    private fun determineDefault(annotations: Collection<AnnotationData>): String? {
         return annotations
             .filter { it.name == Default::class.qualifiedName }
-            .map { it.values["default"] as String }
+            .map { it.values["value"] as String }
             .firstOrNull()
     }
 

